modular_skyrat/modules/ashwalkers/code/buildings/railroad.dm

```diff
@@ -107,34 +104,14 @@
 	return FALSE
 
 /obj/vehicle/ridden/rail_cart/AltClick(mob/user)
-	. = ..()
+	to_chat(user, span_notice("You attempt to attach a trailer to [src]..."))
```

Comment:
is attach_trailer a proc that uses an action bar?

Reply:
no